src/main/java/jpadb/objects/JpaCompany.java

```diff
@@ -0,0 +1,75 @@
+package jpadb.objects;
+
+
+import lombok.extern.log4j.Log4j;
+import jpadb.entities.Company;
+import javax.persistence.EntityManager;
+
+@Log4j
+public class JpaCompany {
```

Comment:
Please write an interface with contract and this class should implement the contract